spec/registry.xml

```diff
@@ -0,0 +1,250 @@
+        "You invited A, B, and 2 more guests."
+      -->
+      <option name="offset" values="positiveInteger"/>
+      <!-- The minimum number of integer digits to use.
+        A value with a smaller number of integer digits than this number will be
+        left-padded with zeros (to the specified length) when formatted.
+      -->
+      <option name="minimumIntegerDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The minimum number of fraction digits to use.
+        The default for plain number and percent formatting is 0;
+        the default for currency formatting is the number of minor unit digits provided by
+        the ISO 4217 currency code list (2 if the list doesn't provide that information).
+      -->
+      <option name="minimumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The maximum number of fraction digits to use.
+        The default for plain number formatting is the larger of minimumFractionDigits and 3;
+        the default for currency formatting is the larger of minimumFractionDigits and the number of minor
+        unit digits provided by the ISO 4217 currency code list (2 if the list doesn't provide that information);
+        the default for percent formatting is the larger of minimumFractionDigits and 0.
+      -->
+      <option name="maximumFractionDigits" values="0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20"/>
+      <!-- The minimum number of significant digits to use. -->
+      <option name="minimumSignificantDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="1"/>
+      <!-- The maximum number of significant digits to use. -->
+      <option name="maximumSignificantDigits" values="1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21" default="21"/>
+
+      <match pattern="anyNumber"/>
+      <match values="zero one two few many"/>
+
+    </matchSignature>
+  </function>
+
+  <function name="select">
+    <description>Literal selection</description>
+    <matchSignature>
+      <match pattern="anythingNotEmpty"/>
+    </matchSignature>
+  </function>
+
+  <!-- ICU does not have this, and one might argue it is just syntactic sugar on top of "select".
+    ICU rejected this a few times.
+  -->
+  <function name="gender">
+    <description>Gender selection</description>
+    <matchSignature>
+      <match values="feminine masculine"/>
+    </matchSignature>
+  </function>
+
+  <!-- TODO: IGNORED for now are the functions that are ICU only or ECMAScript only. To discuss?
+    list : ECMAScripy only
+      https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat
+    relativeTime : ECMAScripy only
+      https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat
+    spellout : ICU only
+    duration : ICU only
+  -->
```

Comment:
Please raise these as separate issue(s) or PR(s) instead.

```suggestion
```

Reply:
Removed, created issue https://github.com/unicode-org/message-format-wg/issues/422 for this (and more)